packages/calcite-components/src/components/list-item/list-item.e2e.ts

```diff
@@ -474,7 +474,7 @@ describe("calcite-list-item", () => {
         {
           "--calcite-list-selection-border-color": {
             shadowSelector: `.${CSS.container}`,
-            targetProp: "borderInlineStartColor",
+            targetProp: "boxShadow",
```

Comment:
Is it weird to be using a box shadow for a border var? Are we breaking any trust with an end user by saying the var is for something but using it for something else? cc @ashetland @alisonailea

Reply:
In this case we opted to use an inner shadow for the selection border styling to maintain alignment with item-group headings, Panel headings, etc. I believe we did something similar for Tile. Defer the @alisonailea for var naming.

Reply:
We do this in Tile, and based on discussions I've had with others on the team, this is an acceptable use case since `box-shadow` doesn't encroach on the dimensions of the box, which allows other sizing properties to not have to accommodate whether the border is there or not.